File: Naloga 34/naloga_34.py
def naloga_34():
    while True:
        znak = input("vpisi znak: ")
        if znak == "&":
            break
        elif znak.islower():
            print(znak)
        else:
            print(ord(znak))
    return
if __name__ == '__main__':
    naloga_34()    

File: Naloga 34/naloga_34.cpp
#include <iostream>

void naloga_34 () {
    while (true) {
        char znak;
        std::cin >> znak;
        if ((int)znak == 38) {
            break;
        }
        else if (islower(znak)) {
            std::cout << znak << std::endl;
        }
        else {
            std::cout << (int)znak << std::endl;
        };
    };
};

int main () {
    naloga_34();
    return 0;
};
